packages/vite-plugin-cloudflare/src/deploy-config.ts

```diff
@@ -0,0 +1,103 @@
+import assert from 'node:assert';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import * as vite from 'vite';
+import type { ResolvedPluginConfig } from './plugin-config';
+
+interface DeployConfig {
+	configPath: string;
+	auxiliaryWorkers: Array<{ configPath: string }>;
+}
+
+function getDeployConfigPath(root: string) {
+	return path.resolve(root, '.wrangler', 'deploy', 'config.json');
+}
+
+export function getConfigPaths(root: string) {
+	const deployConfigPath = getDeployConfigPath(root);
+	const deployConfig = JSON.parse(
+		fs.readFileSync(deployConfigPath, 'utf-8'),
+	) as DeployConfig;
+
+	return [
+		{ configPath: deployConfig.configPath },
```

Comment:
Just for fun you could actually do:

```suggestion
		deployConfig,
```